packaging/rpm/kepler.spec

```diff
@@ -44,9 +41,13 @@ install -d %{buildroot}%{_bindir}
 install -d %{buildroot}%{_sysconfdir}/kepler/
 
 install -d %{buildroot}/var/lib/kepler/data
+install -d %{buildroot}/var/lib/kepler/bpfassets
+install -d %{buildroot}/etc/kepler/kepler.config
 
 install -p -m755 ./_output/kepler  %{buildroot}%{_bindir}/kepler
 install -p -m644 ./packaging/rpm/kepler.service %{buildroot}%{_unitdir}/kepler.service
+install -p -m644 ./bpfassets/libbpf/bpf.o/amd64_kepler.bpf.o %{buildroot}/var/lib/kepler/bpfassets/amd64_kepler.bpf.o
+install -p -m644 ./_output/ENABLE_PROCESS_METRICS %{buildroot}/etc/kepler/kepler.config/ENABLE_PROCESS_METRICS
```

Comment:
how does user of this rpm add more config params, as we add from operator?

Reply:
maybe we need general process for folder  `/etc/kepler/kepler.config` or we just doc this 
otherwise maintain all those config item one by one seems overkill